libproxy/modules/config_kde.cpp

```diff
@@ -19,13 +19,19 @@
  ******************************************************************************/
 
 #include <sys/stat.h>
-#include <unistd.h>
```

Comment:
This would not have worked for Windows at all without your changes, correct? Does removing this and adding the WIN32 #ifdef below break things for other platforms now?

Reply:
It was meant to be used in windows but with the mingw runtime, rather than the msvc runtime. Since I didn't want to deal with those mapping in our deployment, I proposed this fix.

Reply:
That makes sense. I didn't really know what mingw was, that it was basically GNU for Windows, or that it could replace the normal msvc runtime.